components/brave_shields/browser/ad_block_component_filters_provider.cc

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_shields/browser/ad_block_component_filters_provider.h"
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "base/files/file_path.h"
+#include "base/logging.h"
+#include "base/task/thread_pool.h"
+#include "brave/components/brave_shields/browser/ad_block_component_installer.h"
+#include "brave/components/brave_shields/browser/filter_list_catalog_entry.h"
+#include "components/component_updater/component_updater_service.h"
+#include "content/public/browser/browser_task_traits.h"
+
+#define LIST_FILE "list.txt"
```

Comment:
why a define for this and not a constexpr?

Reply:
that's how it was before, but good opportunity to update it

Reply:
Updated this one and the `regional_catalog.json` one as well